src-ui/lib/components/BooksTable.svelte

```diff
@@ -83,7 +89,7 @@
     </thead>
     <tbody>
       {#each booksStore.value as book}
-        <BooksTableRow {book} {handleEdit} />
+        <BooksTableRow {book} {handleEdit} {removeBook} />
```

Comment:
I wonder if `handleEdit` and `removeBook` shouldn't move inside of `BooksTableRow`? Part of the point of having a shared state for the `BooksStore` is that any component can use it directly and affect other components.